mender-update/daemon/context.hpp

```diff
@@ -145,8 +151,10 @@ class Context {
 	mender::update::context::MenderContext &mender_context;
 	events::EventLoop &event_loop;
 
+	auth::Authenticator authenticator;
```

Comment:
Only slightly related -- shouldn't this and some of the other members above and below be `private`? I don't think any code should be allowed to do `context.authenticator.Something` on a `Context` instance.

Reply:
Yes, I suppose you're right. Most members are meant to be used directly, but I guess this is an exception since it's just a prerequisite for one of the other members. I can follow up on this.

Reply:
I think it only applies to `Authenticator` though. All the others are used at some point.